apistructs/issue_state.go

```diff
@@ -135,3 +135,15 @@ type IssueStateNameGetResponse struct {
 	Header
 	Data []IssueStatus `json:"data"`
 }
+
+type IssueStateInitData struct {
+	IssueType IssueType        `json:"issueType"`
+	States    []IssueStateData `json:"states"`
+}
+
+type IssueStateData struct {
+	Index            int64  `json:"index"`
+	Name             string `json:"name"`
+	IssueStateBelong `json:"issueStateBelong"`
+	Relations        []int `json:"relations"`
```

Comment:
is `LinkTo` or `LinkTos` better?

Reply:
adjusted to Links, use `from` `to` define relation
![image](https://user-images.githubusercontent.com/30427474/151322037-6cab0728-3f39-4664-96d1-771aa70dbfed.png)